Automattic/pocket-casts-android
# Improve video sharing cache

## Description

This is an improvement to caching for clip sharing. Previously cache didn't take into account the shared card. This could result in wrong video background being shared because we used cached one.

## Testing Instructions

1. Share a clip as a a video.
2. Without closing the sharing screen edit the clip and select a different card.
3. Share again as a video.
4. You should see a card selected in the 2nd step as a video background.

## Checklist
 
#### I have tested any UI changes...
